src/pybind/mgr/alerts/__init__.py

```diff
@@ -1 +1,5 @@
 from .module import Alerts
+
+__all__ = [
+    'Alerts',
+]
```

Comment:
we need `__all__` if we are a library and our users are importing `from alerts import *`. I would add `# flake8: noqa` at the top of the file